client/driver/logs_test.go

```diff
@@ -0,0 +1,233 @@
+package driver
+
+import (
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+var (
+	logger = log.New(os.Stdout, "", log.LstdFlags)
+	path   = "/tmp/logrotator"
+)
+
+func TestLogRotator_IncorrectPath(t *testing.T) {
+	incorrectPath := "/foo"
```

Comment:
incorrect makes it seem like you are expecting it to be something. I would name it emptyPath or invalidPath